docs/index.rst

```diff
@@ -1,11 +1,42 @@
 OCDS Kingfisher tool
 ====================
 
-OCDS Kingfisher is a tool for downloading, storing and analysing data from publishers of the Open Contracting Data Standard.
+OCDS Kingfisher is a suite of tools for downloading, storing and analysing data from publishers of the Open Contracting Data Standard.
 
+Overview
+========
 
-.. toctree::
+Kingfisher comprises three tools, which integrate to form an OCDS data pipeline:
+
+- `kingfisher-scrape <https://github.com/open-contracting/kingfisher-scrape>`_ , for downloading data from public sources of OCDS data
+- `kingfisher-process <https://github.com/open-contracting/kingfisher-process>`_, for storing OCDS data, and performing operations on it, including transformation and validation. 
+- A suite of queries that can be used to meaningfully interpret OCDS data sets
+
+Hosted Kingfisher
+=================
+
+Kingfisher has been developed primarily for internal use by Open Contracting Partnership, and OCP maintains a hosted instance. If you work or OCP or are part of the OCDS Team, see `Hosted Kingfisher <https://ocdskingfisher.readthedocs.io/en/add-overview-dos/hosted.html>`_ for details of the server, and how to access it. 
```

Comment:
This path is wrong. It should just be a local reference, no?

Reply:
Good spot. Updated.